src/bootstrap/bootstrap.py

```diff
@@ -478,16 +480,19 @@ def bin_root(self):
 
         >>> rb = RustBuild()
         >>> rb.build_dir = "build"
-        >>> rb.bin_root() == os.path.join("build", "stage0")
+        >>> rb.bin_root() == os.path.join("build", "base")
         True
 
         When the 'build' property is given should be a nested directory:
 
         >>> rb.build = "devel"
-        >>> rb.bin_root() == os.path.join("build", "devel", "stage0")
+        >>> rb.bin_root() == os.path.join("build", "base", "devel")
         True
         """
-        return os.path.join(self.build_dir, self.build, "stage0")
+        if self.build == "":
+            return os.path.join(self.build_dir, "base")
```

Comment:
Can you describe why this was changed?

Reply:
It makes sense to download and extract the bootstrap compiler in a separate location from the build artifacts. I also changed to location to avoid having existing code mess with it.